js/gantt-helper.js

```diff
@@ -29,13 +29,15 @@
 /* global gantt */
 /* global displayAjaxMessageAfterRedirect */
 
-var GlpiGantt = (function() {
-
-    var readonly = false;
-    var plugin_path = CFG_GLPI.root_doc + '/' + (GLPI_PLUGINS_PATH['gantt'] ?? "");
-    var url = plugin_path +  '/ajax/gantt.php';
-    var parseDateFormat = "%Y-%m-%d %H:%i";
-    var uiDateFormat = null;
+//TODO: This should be a class
```

Comment:
if you want to go further, don't hesitate.
The code was done by an external contributor (Attila Kotz) and i think it's unlikely he will contribute in the future.

Reply:
I didn't want to spend too much time making changes right away and wanted to avoid possibly adding bugs with refactors. If it does get converted to a class, it would make it easier to implement tests.